src/mongo/db/concurrency/lock_manager.cpp

```diff
@@ -229,7 +229,7 @@ struct LockHead {
 
     // Methods to maintain the granted queue
     void incGrantedModeCount(LockMode mode) {
-        invariant(grantedCounts[mode] >= 0);
+        //invariant(grantedCounts[mode] >= 0);
```

Comment:
Why did you remove this invariant? Do you believe it will never fire? If you can prove that, just delete it, don't comment it out.